app/views/super_admin/organisations/_cba_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<% cba_enabled = organisation.cba_enabled %>
+
+  <p class="govuk-body">
+    Show both EAP/TLS and MS-CHAPv2 authentication attempts in the logs. This is for organisations using Certificate Based Authentication (CBA)
+  </p>
+  <% if cba_enabled %>
+    <p class="govuk-body">
+      This organisation can see the CBA logs.
```

Comment:
I think we can use the Flag for everything CBA related so uploading and managing certificates as well. So that organisations that are not using CBA aren't getting confused by our interface changing. So maybe change it to 'this organisation has CBA enabled' or something